src/routes/home.ts

```diff
@@ -7,6 +7,12 @@ const plugin: FastifyPluginAsyncTypebox = async (fastify) => {
   fastify.get(
     '/',
     {
+      config: {
```

Comment:
@jean-michelet 

I have changed the configuration of the rate limit in `home` because the environment variable `RATE_LIMIT_MAX` was recently introduced with a value of 10 by default, certainly for one of the Tasks tests which fails if we set it to 3.

3 is the default value for the rate limit at the plugin level